server/core/src/main/java/dev/slimevr/tracking/trackers/udp/UDPProtocolParser.kt

```diff
@@ -40,6 +40,25 @@ class UDPProtocolParser {
 				buf.position(bundlePacketStart + bundlePacketLen)
 			}
 			return bundlePackets.toTypedArray()
+		} else if (packetId == PACKET_BUNDLE_V2) {
```

Comment:
These too please, for consistency with the flag docs.
```suggestion
		} else if (packetId == PACKET_BUNDLE_COMPACT) {
```